backend/__init__.py

```diff
@@ -151,14 +151,14 @@ def to_dict(self):
 
 class SavedSearch(EmbeddedDocument):
     saved_search_id = DB.ObjectIdField(default=ObjectId)
-    metadata = DB.StringField(max_length=4096)
-    query = DB.StringField(max_length=4096)
+    metadata = DB.StringField()
+    query = DB.StringField()
     count = DB.LongField()
 
     def to_dict(self):
         dic = {}
         dic['id'] = str(self.saved_search_id)
-        dic['meta'] = json.loads(str(self.metadata))
+        dic['query'] = json.loads(str(self.metadata))
```

Comment:
any particular reason why we don't return 'meta' and 'query' seperately instead of returning metadata as 'query'?